draft-ietf-quic-transport.md

```diff
@@ -1005,18 +1005,17 @@ Packet numbers are divided into 3 spaces in QUIC:
 - Application data space: All 0-RTT and 1-RTT encrypted packets
   {{packet-protected}} are in this space.
 
-As described in {{QUIC-TLS}}, each packet type uses different encryption keys.
+As described in {{QUIC-TLS}}, each packet type uses different protection keys.
 
-Conceptually, a packet number space is the encryption context in which
-a packet can be processed and ACKed.  Initial packets can only be sent
-with Initial encryption keys and ACKed in packets which are also
-Initial packets.  Similarly, Handshake packets can only be sent and
-acknowledged in Handshake packets.
+Conceptually, a packet number space is the context in which a packet can be
+processed and ACKed.  Initial packets can only be sent with Initial encryption
+keys and ACKed in packets which are also Initial packets.  Similarly, Handshake
+packets can only be sent and acknowledged in Handshake packets.
```

Comment:
"Handshake packets can only be sent in Handshake packets" is either tautological or incorrect, and I'm uncertain which.  Either way, I don't think this says what you mean to say.